source/atlas-cli-local-cloud.txt

```diff
@@ -37,9 +37,9 @@ actions:
    manage that deployment with the {+atlas-cli+}.
 
 You can use only ``atlas deployments`` {+atlas-cli+} commands with 
-local |service| deployments. You can use all {+atlas-cli+} commands, 
-including ``atlas deployments`` commands,  with cloud |service| 
-deployments.
+local |service| deployments. You can use all other {+atlas-cli+} 
+commands, including ``atlas deployments`` commands,  with cloud 
+|service| deployments.
```

Comment:
I'm still grappling with this statement a bit, I think it was more accurate without adding "other" because the options are:

- local deployments: can only use deployments commands
- cloud deployments: can use all commands (or, stated another way, can use both deployments commands AND all other commands, but "all other commands" excludes deployment commands). 

The statement "all **other** commands, including atlas deployments" seems to both include and exclude deployments commands-- it should either be "all commands, including atlas deployments" or "all other commands" with no mention of deployments (because "other" means deployments commands are excluded from this statement)